tools/generate_api.py

```
@@ -510,49 +755,250 @@ def generate_terraform(endpoints: List[Dict[str, Any]]) -> None:
+            
+            # Request body
+            request_body = operation.get('requestBody', {})
+            if request_body:
+                lines.append("**Request Body:**")
+                lines.append("")
+                content = request_body.get('content', {})
+                if 'application/json' in content:
+                    schema = content['application/json'].get('schema', {})
+                    if '$ref' in schema:
+                        schema_name = schema['$ref'].split('/')[-1]
+                        lines.append(f"```json")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)